Branch-SDK-Automation-TestBed/src/main/res/values/strings.xml

```diff
@@ -273,4 +273,47 @@
         <item>CLICK_AD</item>
         <item>VIEW_AD</item>
     </string-array>
+    <string name="title_activity_display_screen">DisplayScreen</string>
+    <!-- Strings used for fragments for navigation -->
+    <string name="first_fragment_label">First Fragment</string>
+    <string name="second_fragment_label">Second Fragment</string>
+    <string name="next">Next</string>
+    <string name="previous">Previous</string>
+
+    <string name="lorem_ipsum">
```

Comment:
Is this string used? Couldn't find any reference

Reply:
All strings removed.